contracts/sources/lending_market.move

```diff
@@ -379,6 +381,26 @@ module suilend::lending_market {
         obligation::zero_out_rewards_if_looped(obligation, &mut lending_market.reserves, clock);
         coin::from_balance(receive_balance, ctx)
     }
+    
+    /// Set emode for obligation - T is the deposit coin type
+    public fun set_emode<P>(
```

Comment:
we also need the ability to unset emode. maybe call this function `set_emode_flag', which also accepts a boolean which indicates whether the obligation is emode or not